--- lib/ExtUtils/MM_Unix.pm~	2003-09-15 22:14:42.000000000 +0000
+++ lib/ExtUtils/MM_Unix.pm	2003-10-21 21:13:59.000000000 +0000
@@ -2052,7 +2052,7 @@
 
         unless ($Config{$k}) {
             $Config_Override{$k} = $Config{usevendorprefix} ?
-                  $self->catdir($Config{vendorprefixexp}, 'man', "man$num") :
+                  $self->catdir($Config{vendorprefixexp}, 'share', 'man', "man$num") :
                   '';
         }
     }
@@ -2069,12 +2069,12 @@
     # it up.
     unless( $Config{installsiteman1dir} ) {
         $Config_Override{installsiteman1dir} = 
-          $self->catdir($sprefix, 'man', 'man1');
+          $self->catdir($sprefix, 'share', 'man', 'man1');
     }
 
     unless( $Config{installsiteman3dir} ) {
         $Config_Override{installsiteman3dir} = 
-          $self->catdir($sprefix, 'man', 'man3');
+          $self->catdir($sprefix, 'share', 'man', 'man3');
     }
 
     unless( $Config{installsitebin} ) {
