diff -Nur ExtUtils-MakeMaker-6.30.orig/lib/ExtUtils/Install.pm ExtUtils-MakeMaker-6.30.write-permissions/lib/ExtUtils/Install.pm
--- ExtUtils-MakeMaker-6.30.orig/lib/ExtUtils/Install.pm	2005-03-22 04:30:24.000000000 +0000
+++ ExtUtils-MakeMaker-6.30.write-permissions/lib/ExtUtils/Install.pm	2006-10-30 16:14:28.000000000 +0000
@@ -180,7 +180,7 @@
 		print "Installing $targetfile\n";
 		utime($atime,$mtime + $Is_VMS,$targetfile) unless $nonono>1;
 		print "utime($atime,$mtime,$targetfile)\n" if $verbose>1;
-		$mode = 0444 | ( $mode & 0111 ? 0111 : 0 );
+		$mode = 0644 | ( $mode & 0111 ? 0111 : 0 );
 		chmod $mode, $targetfile;
 		print "chmod($mode, $targetfile)\n" if $verbose>1;
 	    } else {
@@ -451,7 +451,7 @@
 	}
 	my($mode,$atime,$mtime) = (stat $from)[2,8,9];
 	utime($atime,$mtime+$Is_VMS,$to);
-	chmod(0444 | ( $mode & 0111 ? 0111 : 0 ),$to);
+	chmod(0644 | ( $mode & 0111 ? 0111 : 0 ),$to);
 	next unless $from =~ /\.pm$/;
 	_autosplit($to,$autodir);
     }
