--- lib/ExtUtils/MM_Unix.pm~	2004-12-09 23:26:02.967520872 +0100
+++ lib/ExtUtils/MM_Unix.pm	2004-12-09 23:28:37.086091288 +0100
@@ -1388,7 +1388,7 @@
     my($name, %dir, %xs, %c, %h, %ignore, %pl_files, %manifypods);
     my %pm;
 
-    @ignore{qw(Makefile.PL test.pl t)} = (1,1,1);
+    @ignore{qw(Makefile.PL test.pl t Build.PL)} = (1,1,1,1);
 
     # ignore the distdir
     $Is_VMS ? $ignore{"$self->{DISTVNAME}.dir"} = 1
