--- ExtUtils-MakeMaker-6.48/inc/ExtUtils/Install.pm.orig	2008-10-10 04:28:25.000000000 +0200
+++ ExtUtils-MakeMaker-6.48/inc/ExtUtils/Install.pm	2009-03-02 14:51:47.297088533 +0100
@@ -780,7 +780,7 @@
                 utime($atime,$mtime + $Is_VMS,$targetfile) unless $dry_run>1;
     
     
-                $mode = 0444 | ( $mode & 0111 ? 0111 : 0 );
+                $mode = 0644 | ( $mode & 0111 ? 0111 : 0 );
                 $mode = $mode | 0222
                     if $realtarget ne $targetfile;
                 _chmod( $mode, $targetfile, $verbose );
@@ -1174,7 +1174,7 @@
         }
         my($mode,$atime,$mtime) = (stat $from)[2,8,9];
         utime($atime,$mtime+$Is_VMS,$to);
-        _chmod(0444 | ( $mode & 0111 ? 0111 : 0 ),$to);
+        _chmod(0644 | ( $mode & 0111 ? 0111 : 0 ),$to);
         next unless $from =~ /\.pm$/;
         _autosplit($to,$autodir);
     }
